client/src/client.js

```diff
@@ -4,12 +4,11 @@ const ecc = require('eosjs-ecc')
 const semver = require('semver');
 const NodeCache = require("node-cache");
 const hash = require('object-hash');
-const { Orejs } = require('@open-rights-exchange/orejs')
-const URL = require('url').URL
+const {
```

Comment:
Please change your linter settings (and save them to the project) so it doesn't do this - we don't want variables like in this destructuring to be spread across multiple lines.